### ISSUE TITLE
Meaning of asterisk is not explained at the beginning of the form - All form pages with asterisks

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#16]
**URL**: All - eg. https://hcdev.openplus.ca/en/ 
**User Agent**: Firefox, Edge
**Language**: Both

**Issue Description**: The required status of the form fields are indicated using an asterisk in field label. However, the meaning of the asterisk is not defined at the start of the form.  

**Expected results**: The meaning of the asterisk should be defined at the start of the form.

Please follow WET format to include the text "required" in the label https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html

**Comment by mmiron-HC** [comment]
This affects the Login and Reset password pages as well.

**Comment by davidgotlieb** [comment]
﻿This is not as simple as it sounds, there is an issue with Drupal core. Work is underway to fix it within the Core. Not recommended to address within individual templates.

**Comment by echouin** [comment]
Hi @mmiron-HC Can we close this one? This is a WxT / Drupal core issue. There is an issue for it in Drupal. This is tagged as moderate, so I ﻿assume not a deal breaker.

**Comment by mmiron-HC** [comment]
@echouin no we can't close these as they are still valid issues. Mark it using a milestone. All WCAG issues must be fixed eventually. We need to explain to management/CIO what risk they are accepting. We will identify all issues that are outstanding and explain that some are out of your control (platform) and others are still in your hands (if there are any outstanding). But in the end, all WCAG issues regardless of the severity should be addressed.

**Comment by echouin** [comment]
Hello @mmiron-HC I believe that this has been addressed. Please see the subscription page: https://hcdev.openplus.ca/en/form/subscribe

**Comment by mmiron-HC** [comment]
@echouin no it hasn't.

**Comment by echouin** [comment]
@mmiron-HC fixed.

**Comment by hydegeoff** [comment]
Issue can be closed
